LIBRARIES.md

```diff
@@ -0,0 +1,153 @@
+# Module libraries
+
+The Toolkit can import modules from external libraries.
+
+## Usage
+
+A user will add the library url to cdf.toml:
+
+```toml
+[[libraries]]
+url = "https://example.com/my-library/<root folder>"
+```
+
+## Publishing a library
+
+The library must be available with https. Authentication is not currently supported.
+
+To publish a library, create a repository with the following structure. It is strictly required for the
+Toolkit to recognize it and present it to the user in the cdf modules init or cdf modules add commands.
+
+```example:
+
+<root folder>
+├── package.toml
```

Comment:
This is sufficient for us to run the entire `cdf modules init/add` command, and only at the very end start the download process of what the user selects?